Competition2017/src/competition/subsystems/RobotSide.java

```diff
@@ -0,0 +1,6 @@
+package competition.subsystems;
```

Comment:
It's reasonable question, I can see arguments both ways. On the one hand it isn't a Subsystem but on the other it is instrumentally about Subsystems. For instance we have the `SubsystemDefaultCommandMap` in the top level subsystems package as well.